tests/integration/testlib/env.py

```diff
@@ -38,3 +38,11 @@ def is_k8s():
 
 def is_el8():
     return exec_cmd("rpm -E %{?rhel}".split())[1].strip() == "8"
+
+
+def nm_libreswan_micro_version():
+    return int(
+        exec_cmd("rpm -q NetworkManager-libreswan --qf %{VERSION}".split())[1]
+        .split(".")[-1]
+        .strip()
+    )
```

Comment:
Just checking the micro version is not enough. When we bump to NM-libreswan 1.24.0, micro < 20 will be true, so the failures in the tests will be masked. Better to do a full version comparison.

Reply:
Fixed by `nm_libreswan_version_int() < version_str_to_int("1.2.22")`